ChangeLog.md

```diff
@@ -23,6 +23,10 @@ Current Trunk
    ad hoc constructed rules around default Emscripten uses. The old behavior
    will be deprecated and removed in the future. Build with -s ASSERTIONS=1
    to get diagnostics messages related to this transition.
+	
+ - Option -s EMTERPRETIFY_WHITELIST now accepts shell-style wildcards;
```

Comment:
better to add new stuff on the top, as then when new releases are added, the new thing would not be at risk of being included by mistake in the wrong release. (I just added 1.38.27 now, so this will need to be moved)